### PR TITLE
Add suggestions to restore and backup

### PR DESCRIPTION
**Pull request by tony-landreth**
PGO-646

### CODE REVIEW

File: internal/cmd/backup.go [hunk 1]
```diff
@@ -128,6 +129,9 @@ postgresclusters/hippo backup initiated`)
 		)
 
 		if err != nil {
+			if strings.Contains(err.Error(), "conflict") {
+				cmd.Printf("SUGGESTION: The --force-conflicts flag may help in performing this operation.")
```
Comment (benjaminjb) [inline internal/cmd/backup.go:133]: "\n" around printf?

**Review by benjaminjb — APPROVED** [review]
LGTM, no blockers, just want to make sure the output is well-spaced

File: internal/cmd/backup.go (hunk 1 above)
Comment (tony-landreth) [reply internal/cmd/backup.go:133]: The error prints on the next line in this case.

File: internal/cmd/restore.go [hunk 2]
```diff
@@ -190,6 +193,9 @@ func (config pgBackRestRestore) Run(ctx context.Context) error {
 		config.PostgresCluster, types.ApplyPatchType, patch,
 		config.Patch.PatchOptions(patchOptions))
 	if err != nil {
+		if strings.Contains(err.Error(), "conflict") {
```
Comment (cbandy) [inline internal/cmd/restore.go:196]: ⛏️ Perhaps this function identifies the error? https://pkg.go.dev/k8s.io/apimachinery/pkg/api/errors#IsConflict

```suggestion
		if apierrors.IsConfict(err) {
```

Comment (benjaminjb) [reply internal/cmd/restore.go:196]: Hmm, yeah, makes sense to me:

https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/util/managedfields/internal/conflict.go#L42
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/api/errors/errors.go#L249

Comment (tony-landreth) [reply internal/cmd/restore.go:196]: Nice! Thanks for the suggestion. Here's a [PR](https://github.com/CrunchyData/postgres-operator-client/pull/82).

**Pull request merged**